client/src/templates/layouts/Main.js

```diff
@@ -3,7 +3,7 @@ import { theme, mq } from "constants/theme";
 const { offWhite } = theme.colors;
 
 const Main = styled.main`
-  padding: 0 2rem;
+  padding: 7rem 2rem 0rem 2rem;
```

Comment:
@robinv85 do we need this much padding though? I am actually deleting this padding in another PR. I had a picture that I want to display from left to right completely and this causes to padding always be 20px on the left and right... Can we remove this @robinv85 ?

Reply:
@djevm I don't know, is that to provide padding where the nav manu will come? have no idea, would have to check the code and see the elements in the browser

Reply:
So what happens here is that this padding it is applied to all the main part of the page, except the footer and the navbar. I had some trouble actually displaying an image on width: 100% when I was working on the About Us page cuz this damn 2rem padding on left and right. I believe doing this will provide padding from the navbar with the main page content, and again 2 rem left and 2 rem right. I would much rather not have this since as my example of wanting to display an image full width, there was not an easy way to actually override this since everything would be a child of main. I think if one's wants padding on a specific page, they should add it on their container rather than applying it across the platform.

Reply:
Since we have sections that must be full width, I think we should avoid using this padding in a container that holds all the other components.

Reply:
That's correct. I agree with you. I have already removed this property in another PR, so perhaps we should keep it removed. If padding is needed, then add padding to the header or the component that needs padding?

Reply:
But yeah @robinv85, I guess to better explain it as I didn't do really well on the last one. This setting applies to the main component that holds all the components that are on the page, so as Joeu said, if you had a component that you want to display in full width, then this padding on the main will give you a hard time since it is the parent of that component.

Reply:
Are we okay with keeping just the top and bottom paddings on main? @Joeu @idjevm

Reply:
@jyclam , so far I've just hit the need of change for left and right padding when @idjevm and I discussed about the AboutUs page. So I think it is ok to keep top and bottom paddings, after all we have header and footer across pages.

Reply:
yeah now I understand, seems better not to add this here indeed
so the padding at the top can stay? 
But maybe also only add that inside the layout, not on the full Main component here
in fact the layout templates structure also needs some work i think

Reply:
@robinv85 we should probably do the layout templates rework in another PR. Is there a better place to put this top padding with the current layouts?

Reply:
Let's just remove that padding completely. If there are any issues, let's add padding to the needed container.

Reply:
Ah, okay. Just saw this comment. I can go through and see which pages will need to be padded

Reply:
I mean, can't we add it inside the `NavigationLayout` component only? 
Just because we could have a "main" element also in another layout without the main navigation so it might have different needs, but I have opened an issue to rework the layout templates  as well